scripts/codegen.js

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env node
+
+const { join } = require('path');
+const telescope = require('@osmonauts/telescope').default;
+
+telescope({
+    protoDirs: [
+        'wasmd-0.25/proto',
+        'wasmd-0.25/third_party/proto',
+        'cosmos-sdk-0.45/proto',
+        'cosmos-sdk-0.45/third_party/proto'
+    ],
+    outPath: join(__dirname, '/../src'),
+    options: {
+        includePackageVar: true,
+        excluded: {
+            protos: [
+                'cosmos/authz/v1beta1/event.proto'
+            ]
+        },
+        typingsFormat: {
+            useExact: true,
+            timestamp: 'timestamp',
+            duration: 'duration'
+        },
+        lcdClients: {
+            enabled: false
+        },
+        rpcClients: {
+            enabled: true,
+            bundle: false,
+            camelCase: false
+        },
+        aminoEncoding: {
+            enabled: false
+        }
+    }
+}).then(() => {
+    console.log('✨ All Done!');
+})
```

Comment:
Shouldn't we also catch errors, log them and return a non-zero code? I typically do something like:

```
.catch((error) => {
    console.error(error);
    process.exit(1);
  });
```